src/invert.js

```diff
@@ -12,7 +12,7 @@ var keys = require('./keys');
  * @since v0.9.0
  * @category Object
  * @sig {s: x} -> {x: [ s, ... ]}
- * @param {Object} obj The object or array to invert
+ * @param {Object|*} obj The object or array to invert
```

Comment:
My reasoning were these tests which seemed to me like we're specifically supporting those kinds of values. But I agree with you that `*` doesn't make sense for `invert` or `invertObj`.

```js
  it('returns an empty object when applied to a primitive', function() {
    eq(R.invert(42), {});
    eq(R.invert('abc'), {});
  });

  it('returns an empty object when applied to null/undefined', function() {
    eq(R.invert(null), {});
    eq(R.invert(undefined), {});
  });
```